tutor/commands/upgrade/compose.py

```diff
@@ -46,6 +46,9 @@ def upgrade_from(context: click.Context, from_release: str) -> None:
     if running_release == "palm":
         running_release = "quince"
 
+    if running_release == "quince":
+        upgrade_from_quince(context, config)
+
```

Comment:
Will Nightly users be able to upgrade by running `tutor ... upgrade --from=quince` ? If so, shall we send that message out to #tutor-maintenance once this merges.?

Reply:
Yes 👍🏽